package.json

```
@@ -59,7 +59,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^2.8.8",
     "remark-cli": "^11.0.0",
-    "stylelint": "^16.0.0"
+    "stylelint": "github:stylelint/stylelint"
   },
   "peerDependencies": {
     "stylelint": "^16.0.0"
```

```suggestion
    "stylelint": "^16.1.0"
  },
  "peerDependencies": {
    "stylelint": "^16.1.0"
```

Then run to update `package-lock.json`:

```sh
npm i
```